source/form.html

```diff
@@ -15,28 +15,81 @@
     </a>
     <nav class="main-nav">
       <button class="main-nav__open-button" type="button">Открыть меню</button>
-      <ul class="main-nav__list cite-list">
-        <li class="cite-list__item">
-          <a class="cite-list__link" href="catalog.html">Каталог товаров</a>
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item">
+          <a class="site-list__link" href="catalog.html">Каталог товаров</a>
         </li>
-        <li class="cite-list__item">
-          <a class="cite-list__link">Вязание на заказ</a>
+        <li class="site-list__item">
+          <a class="site-list__link">Вязание на заказ</a>
         </li>
       </ul>
-      <ul class="main-nav__list cite-list cite-list--user">
-        <li class="cite-list__item">
-          <a class="cite-list__link cite-list__link--search" href="#">Поиск по сайту</a>
+      <ul class="main-nav__list site-list site-list--user">
+        <li class="site-list__item">
+          <a class="site-list__link site-list__link--search" href="#">Поиск по сайту</a>
         </li>
-        <li class="cite-list__item">
-          <a class="cite-list__link cite-list__link--cart" href="#">Корзина: пока пуста</a>
+        <li class="site-list__item">
+          <a class="site-list__link site-list__link--cart" href="#">Корзина: пока пуста</a>
         </li>
       </ul>
     </nav>
   </header>
 
   <main class="page-main">
-    <h1 class="page-main__title">Вязание на заказ</h1>
-
+    <h1 class="visually-hidden">Форма заказа</h1>
+    <p class="page-main__title">Вязание на заказ</p>
+    <p class="page-main__text">Мы будем рады воплотить в жизнь ваши пожелания! Заполните простую форму заказа и мы
+      свяжемся
+      с вами, чтобы уточнить детали.</p>
+    <form class="page-main__form form" action="https://echo.htmlacademy.ru" method="post">
+      <fieldset class="form__product-type fieldset">
```

Comment:
```suggestion
      <fieldset class="form__product-type form__fieldset">
```
особенность формы как блока: что внутри неё всё - её элементы.